### PR TITLE
Intro to Bootstrap

### PR DESCRIPTION
**Pull request by rtillies**
Intro to Bootstrap lesson
* missing learning goals
* missing Check for Understanding

Does this lesson need prep and lab?

### CODE REVIEW

**Comment by memcmahon** [comment]
This mini lesson does not need a prep or lab.  I also don't think we need CFUs since this is a lesson to help students style an application, and that's not really core to the the learning goals of the mod.  I don't think it would hurt to have some, but I think we can consider this 'finished' without a CFU section.

File: module3/lessons/Week5/IntroToBootstrap.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+We will build on our foundational understanding of [HTML](/module3/lessons/week1/HTMLFundamentals) and [CSS](/module3/lessons/week1/CSSFundamentals).
+
+## Warm Up
+
+<section class='call-to-action' markdown='1'>
+
+If necessary, re-familiarize yourself with earlier lessons:
+* [HTML Fundamentals](/module3/lessons/week1/HTMLFundamentals)
+* [CSS Fundamentals](/module3/lessons/week1/CSSFundamentals)
+
```
Comment (memcmahon) [inline module3/lessons/Week5/IntroToBootstrap.md:20]: Should the 'call-to-action' section end here?  It feels a little funky to have this section be the whole lesson.

File: module3/lessons/Week5/IntroToBootstrap.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+## What is Bootstrap?
+
+Bootstrap is a popular CSS Framework used to develop responsive and mobile-first websites, based on a grid design. Most everything is built in a container, with rows and columns nested inside.
+
+Most importantly, it is **FREE** to use.
+
+## How to use Bootstrap
+
+You can use Bootstrap in one of two ways:
```
Comment (memcmahon) [inline module3/lessons/Week5/IntroToBootstrap.md:29]: Is there a preferred method for using bootstrap?  I have always used a CDN - are we going to ask students to try both ways?

File: module3/lessons/Week5/IntroToBootstrap.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+* The fourth row has 3 elements, each element spans 4 columns.
+* The fifth row has 2 elements, each element spans 6 columns.
+* The sixth row also has 2 elements, but each element span a different number of columns (4 and 8, respectively).
+* The last row has 1 element that spans all 12 columns.
+
+_Note:_ You can have exactly 12 columns, or less than 12 columns (the unused columns will appear as empty space). However, you cannot have more than 12 columns.
+
+## Responsiveness
+
+* view the [website](https://launchbootstrap.launch-team.repl.co/)
```
Comment (memcmahon) [inline module3/lessons/Week5/IntroToBootstrap.md:68]: Love these example repls!!

File: module3/lessons/Week5/IntroToBootstrap.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+Some adjustments have been made for the smaller screen, but it remains unchanged for the most part.
+
+Same webpage, this time on my phone:
+![Website Tablet](/assets/images/module3/Week5/OlympicPhone.png)
+
+Whoa, big difference. But it would have been really difficult to read the text and view the images if they were all still side by side. I have to scroll to view the information for Paris (not to mention Beijing), but the experience is much better.
+
+## Classes
+
+The beauty of Bootstrap is that you attach classes to HTML tags, and the magic just... happens.
```
Comment (memcmahon) [inline module3/lessons/Week5/IntroToBootstrap.md:89]: I think we should add some more information about the 'magic'.  Maybe we can make it more clear that we are attaching classes that correspond with pre-existing css styling targets in bootstrap.

File: module3/lessons/Week5/IntroToBootstrap.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+| Extra Small  | (none)      | < 576 px  | Smartphone (portrait)   |
+| Small        | `sm`        | >= 576    | Smartphone (landscape)  |
+| Medium       | `md`        | >= 768    | Tablet |
+| Large        | `lg`        | >= 992    | Laptop screen  |
+| Extra Large  | `xl`        | >= 1200   | Desktop monitor |
+| 2x Large     | `xxl`       | >= 1400   | High resolution monitor |
+
+This explains why the webpage displays columns on the desktop and tablet, but becomes a single column on the phone. In our example, we set our breakpoint to `md` so that it would appear "normal" on tablets and larger devices.
+
+## Want more Bootstrap?
```
Comment (memcmahon) [inline module3/lessons/Week5/IntroToBootstrap.md:156]: Great overview of additional resources :)

**Review by memcmahon — APPROVED** [review]
I think this is just the right amount of content - love the example code!

Could you add a link to this lesson on the Mod3 Index?

File: module3/lessons/Week5/IntroToBootstrap.md (hunk 2 above)
Comment (rtillies) [reply module3/lessons/Week5/IntroToBootstrap.md:29]: I think the CDN is the preferred method and this is how students should use it. Installing Bootstrap is for informational purposes only.

File: module3/lessons/Week5/IntroToBootstrap.md (hunk 4 above)
Comment (rtillies) [reply module3/lessons/Week5/IntroToBootstrap.md:89]: you probably tell kids that Santa Claus doesn't exist, don't you? 😂 j/k

**Pull request merged**